.github/workflows/inspektor-gadget.yml

```diff
@@ -548,6 +548,54 @@ jobs:
       run: |
         make controller-tests
 
+  benchmarks:
+    name: Benchmarks
+    # level: 0
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup go
+      uses: actions/setup-go@v3
+      with:
+        go-version: ${{ env.GO_VERSION }}
+        cache: true
+      id: go
+    - name: Install debian packages
+      uses: ./.github/actions/install-debian-packages
+    - name: Run benchmarks
+      run: go test -exec sudo -bench=. -run=Benchmark ./pkg/gadgets/... ./internal/benchmarks/... | tee output.txt
+    #- name: Download previous benchmark data
+    #  uses: actions/cache@v1
+    #  with:
+    #    path: ./cache
+    #    key: ${{ runner.os }}-benchmark
+    - name: Store benchmark result
+      uses: benchmark-action/github-action-benchmark@v1
+      with:
+        name: Gadget benchmarks
+        # What benchmark tool the output.txt came from
+        tool: 'go'
+        # Where the output from the benchmark tool is stored
+        output-file-path: output.txt
+        # Where the previous data file is stored
+        # external-data-json-path: ./cache/benchmark-data.json
+        # Workflow will fail when an alert happens
+        fail-on-alert: false
+        # GitHub API token to make a commit comment
+        github-token: ${{ secrets.BENCHMARKS_TOKEN }}
```

Comment:
Why can't we use `GITHUB_TOKEN` here directly? Or `BENCHMARKS_TOKEN` offers something else?

Reply:
Ah, makes sense. Thanks for clarifying. The reason I asked was, we could widen the [permission at job level](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/.github/workflows/inspektor-gadget.yml#L227) for inspektor-gadget but it doesn't work for another repository.